### PR TITLE
[14.0] [IMP] sale_blanket_order: add taxes and subtotal columns

### PR DESCRIPTION
**Pull request by renda-dev**
- Changed "Quantity" to "Qty" in tree view
- Added "Taxes" and "Subtotal" fields in tree view

immagine

### CODE REVIEW

**Review by PicchiSeba — APPROVED** [review]
Code review: LGTM

File: sale_blanket_order/views/sale_blanket_order_views.xml [hunk 1]
```diff
@@ -142,10 +142,22 @@
                                     <field name="product_uom" groups="uom.group_uom" />
                                     <field name="price_unit" />
                                     <field name="date_schedule" />
-                                    <field name="ordered_uom_qty" />
-                                    <field name="invoiced_uom_qty" />
-                                    <field name="delivered_uom_qty" />
-                                    <field name="remaining_uom_qty" />
+                                    <field
+                                        name="ordered_uom_qty"
+                                        string="Ordered Qty"
```
Comment (rousseldenis) [inline sale_blanket_order/views/sale_blanket_order_views.xml]: @renda-dev IMHO, string is not needed as defined in field definition

**Review by francesco-ooops — APPROVED** [review]
Functional ok!

@rousseldenis good now?

**Comment by francesco-ooops** [comment]
added also commit for maintainership

**Review by HekkiMelody — APPROVED** [review]
Code review, LGTM

**Pull request merged**